README.md

```
@@ -7,7 +7,7 @@
     </a>
     <h1>ZenStack</h1>
     <a href="https://www.npmjs.com/package/zenstack">
-        <img src="https://img.shields.io/npm/v/zenstack">
+        <img src="https://img.shields.io/npm/v/zenstack/release-v2">
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)